Sources/Widgets/Representations/ArrowHandleRepresentation/index.js

```diff
@@ -22,10 +30,14 @@ function vtkArrowHandleRepresentation(publicAPI, model) {
   // Set our className
   model.classHierarchy.push('vtkArrowHandleRepresentation');
 
+  const superClass = { ...publicAPI };
   // --------------------------------------------------------------------------
   // Internal polydata dataset
   // --------------------------------------------------------------------------
 
+  let firstShape = false;
```

Comment:
get rid of it. it should not be needed. If shapeName is empty it means it is a first shape